Applications/Utils/Tests.cmake

```diff
@@ -69,29 +69,41 @@ AddTest(
 )
 
 AddTest(
-    NAME identifySubdomains_riverTriangleMesh
+    NAME identifySubdomains_PntLineSFC_DomainTRIMesh
     PATH MeshGeoToolsLib/IdentifySubdomains
     EXECUTABLE identifySubdomains
-    EXECUTABLE_ARGS -m river_domain_triangle.vtu -o ${Data_BINARY_DIR}/MeshGeoToolsLib/IdentifySubdomains/triangle_ -- river_bc.vtu
+    EXECUTABLE_ARGS -m PntLineSFC_DomainTRIMesh.vtu -o ${Data_BINARY_DIR}/MeshGeoToolsLib/IdentifySubdomains/TRI_ -- PntLineSFC_BC_Point.vtu PntLineSFC_BC_Line.vtu PntLineSFC_BC_Surface.vtu
```

Comment:
Same here; why not `triangle_mesh.vtu` and `point.vtu`, `line.vtu`, `surface.vtu`?
Keep in mind, other people have to read the text many times---readability is important.

Reply:
Thanks, I will rename it again.